aries-site/src/examples/templates/wizard/SimpleWizardExample.js

```diff
@@ -0,0 +1,289 @@
+  const [formValues, setFormValues] = useState({});
+
+  // controls state of cancel layer
+  const [open, setOpen] = useState(false);
+  const size = useContext(ResponsiveContext);
+
+  return (
+    <>
+      <Box width={{ max: 'xxlarge' }} margin="auto" fill>
+        <Header
+          border={{ side: 'bottom', color: 'border-weak' }}
+          pad="small"
+          fill="horizontal"
+          justify="center"
+        >
+          <Box direction="row" flex>
+            {active > 1 && (
+              <Button
+                label={size !== 'small' ? `Step ${active - 1}` : undefined}
+                icon={<FormPreviousLink color="text-strong" />}
+                onClick={() => setActive(active - 1)}
+              />
+            )}
+          </Box>
+          <Box>
+            <Text color="text-strong" weight="bold">
+              Action Title
+            </Text>
+          </Box>
+          <Box direction="row" flex justify="end">
+            <Button
+              label={size !== 'small' ? 'Cancel' : undefined}
+              icon={<FormClose color="text-strong" />}
+              reverse
+              onClick={() => setOpen(true)}
+            />
+          </Box>
+        </Header>
+        <Box
+          align="center"
+          pad={size !== 'small' ? 'large' : 'medium'}
+          flex={false}
+        >
+          <Box width="medium" gap="medium">
+            <Box gap="medium" flex={false}>
+              <Box>
+                <Heading color="text-strong" margin="none">
+                  {steps[active - 1].title || `Step ${active} Title`}
+                </Heading>
+                <Text
+                  color="text-strong"
+                  size="small"
+                >{`Step ${active} of ${steps.length}`}</Text>
+              </Box>
+              <Text color="text-strong" size="large">
+                {/* Index an array starts at 0 */}
+                {steps[active - 1].description}
+              </Text>
+            </Box>
+            <Form
+              value={formValues}
+              onChange={setFormValues}
+              onSubmit={({ value }) => console.log(value)}
+            >
+              {/* Index an array starts at 0 */}
+              {steps[active - 1].inputs(active, setActive)}
+            </Form>
+          </Box>
+        </Box>
+      </Box>
+      {open && <CancellationLayer onSetOpen={setOpen} />}
+    </>
+  );
+};
+
+const CancellationLayer = ({ onSetOpen }) => {
+  return (
+    <Layer
+      position="center"
+      onClickOutside={() => onSetOpen(false)}
+      onEsc={() => onSetOpen(false)}
+    >
+      <Box pad="large" gap="small" width="large">
+        <Box>
+          <Heading color="text-strong" margin="none">
+            Cancel
+          </Heading>
+          <Text color="text-strong">Action Title</Text>
+        </Box>
+        <Text color="text-strong">
+          Cancelling setup will lose all of your progress. Are you sure you want
```

Comment:
was this actually suppose to happen or just example ?

Reply:
okay yeah I wasnt sure if we actually wanted to mock this behavior with state since if you cancel now the values are still there or they will get the idea and have their own state they will update

Reply:
Gotcha, good point. Yeah, wondering how far we want to mock the behavior. My thought is that if you truly "cancelled", the wizard would close and you'd go back to the application wherever you were prior to opening. Will get clarification on this behavior.